.flowconfig

```diff
@@ -33,6 +33,7 @@ node_modules/warning/.*
 .*/node_modules/@react-native-community/cli/.*/.*
 .*/react-native-linear-gradient/.*
 .*/theming/.*
+.*/node_modules/react-native/Libraries/*
```

Comment:
Added this as in the latest version it was scanning react-native

Reply:
What effect does this have?

My assumption is that we want it to scan RN, otherwise when we import things it won't know the type of them, and therefore can't error when we use them incorrectly.

I might be misunderstanding though

Reply:
Before adding this it was running flow against the library and throwing errors on untyped components that we couldn't fix as they are inside react native

Reply:
Will have a double check but I remember seeing some very strange behaviour going on which was invalid which is why I added it

Reply:
Ok cool, yeah I'll try it locally too to see if it has the effect I'm thinking

Reply:
An example - looks like RN has some flow issues itself

![Screenshot 2020-09-28 at 14 47 29](https://user-images.githubusercontent.com/8831547/94440412-97014900-0199-11eb-8db7-42370881cae2.png)

Reply:
Yikes, yeah let's just leave it commented out and not get involved with that 😂